docs/guide/result.md

```diff
@@ -176,17 +176,18 @@ A path can contain integer elements too (when using `Each` rule for example):
 
 #### Resolving special characters collision in attribute names
 
-When attribute name contains value path separator (dot - `.` by default) or `Each` rule shortcut (asterisk -`*`), 
-they're automatically escaped using backslash (`\`) in the error messages list:
+
+When the attribute name contains a path separator (dot - `.` by default) or `Each` rule shortcut (asterisk -`*`), 
```

Comment:
What's the reason of this change: "value path" -> "path"?

Reply:
Because it is "path separator", not "value path separator". It was also suggested by Chris.